### PR TITLE
Add support for dockerize for scalardl test

### PR DESCRIPTION
**Pull request by miseyu**
## Summary
- Add dockernize to `docker`.
- Support setting holderId from config.toml

### CODE REVIEW

**Comment by feeblefakie** [comment]
@miseyu I assume this `holderId` thing is also not needed and removed. Am I correct?

**Comment by miseyu** [comment]
@feeblefakie 
Yes, it is not necessary and has been removed.
https://github.com/scalar-labs/kelpie-test/pull/40/commits/fd4d01a3265b776ad03b5397b3871f4f730aaede

File: scalardl-test/build.gradle [hunk 1]
```diff
@@ -9,10 +9,10 @@ repositories {
 }
 
 dependencies {
-    implementation group: 'com.scalar-labs', name: 'kelpie', version: '1.1.0'
+    implementation group: 'com.scalar-labs', name: 'kelpie', version: '1.2.0'
     implementation group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '3.1.0'
     implementation group: 'com.google.inject', name: 'guice', version: '5.0.1'
-    api group: 'com.scalar-labs', name: 'scalardb', version: '2.4.1'
+    api group: 'com.scalar-labs', name: 'scalardb', version: '3.3.0'
```
Comment (feeblefakie) [inline scalardl-test/build.gradle:15]: It's fine since it's a non-critical kelpie-test repo, but ideally, this kind of upgrade should be done in another PR.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

File: scalardl-test/build.gradle (hunk 1 above)
Comment (miseyu) [reply scalardl-test/build.gradle:15]: I am very sorry. I will be careful. :bow:
